backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -2171,3 +2171,72 @@ databaseChangeLog:
       changes:
         sql: |
           GRANT SELECT ON TABLE ${database.defaultSchemaName}.patient_registration_link TO ${noPhiUsername};
+  - changeSet:
+      id: add-patient-preferences-to-person-view
+      author: nathan@skylight.digital
+      comment: Add person patient preferences to person_no_phi_view
+      changes:
+        - dropView:
+            viewName: person_no_phi_view
+        - createView:
+            viewName: person_no_phi_view
+            remarks: A subset of the person table with columns containing PHI removed.
+            replaceIfExists: true
+            fullDefinition: false
+            selectQuery: |
+              SELECT
+                p.internal_id,
+                p.created_at,
+                p.created_by,
+                p.updated_at,
+                p.updated_by,
+                p.is_deleted,
+                p.organization_id,
+                p.facility_id,
+                p.race,
+                p.gender,
+                p.ethnicity,
+                p.county,
+                p.state,
+                p.employed_in_healthcare,
+                p.role,
+                p.resident_congregate_setting,
+                p.tribal_affiliation,
+                get_census_dob_group(p.birth_date) AS census_dob_group,
+                pp.preferred_language,
+                pp.test_result_delivery
+              FROM ${database.defaultSchemaName}.person p
+              JOIN patient_preferences pp on pp.internal_id=p.internal_id
```

Comment:
Utter nitpick that I don't recommend changing but FYI: in this case you could use `NATURAL JOIN`, which I am glad to see you didn't, but you could also use `JOIN patient_preferences pp USING(internal_id)` for a slightly more readable (IMO) query. It's all syntactical sugar at that point anyway, but I figure if I want people to know about things like that I should tell them. 😄